client/core/core.go

```diff
@@ -1914,6 +1914,7 @@ func (c *Core) loadWallet(dbWallet *db.Wallet) (*xcWallet, error) {
 			}
 		},
 		DataDir: c.assetDataDirectory(assetID),
+		Network: c.Network(),
```

Comment:
See the call to `asset.OpenWallet` about 6 lines down.  Then see:

```go
func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (*ExchangeWallet, error)
```

There is no need to add `Network` to `asset.WalletConfig` because it is provided directly to `NewWallet`.

Reply:
should I just pass `network` directly to `unconnectedWallet()` instead?

like:
`dcr, err := unconnectedWallet(cfg, walletCfg, chainParams, logger, network)`

Reply:
Yeah, or set `dcr.network = network` in `NewWallet`.  I honestly don't care which, but `unconnectedWallet` is probably right.

Also, I don't think the `ExchangeWallet` fields should be exported.